src/storage/local-storage.js

```diff
@@ -9,7 +9,7 @@ class LocalStorage extends Storage {
     localStorage.setItem(key, JSON.stringify(value));
   }
 
-  async delete(key) {
+  async remove(key) {
```

Comment:
why was this renamed?

Reply:
`delete` is a reserved word outside of classes and there are non-class based libraries we may end up using for other functionality within the library that won't be able to use delete

Reply:
AFAIK it's only reserved when used on it's own (e.g. `delete foo.bar`) or as an identifier (e.g. `function delete() { }`).  there's nothing disallowing it from being used as a property name (e.g. `Map.prototype.delete`, `Set.prototype.delete`, etc.)

is your concern we'd import something that would try to `export function delete() { }`?  in that case are you expecting that the API surface of that import would _exactly_ match `Storage`?  or would we have a wrapper (e.g. `class FooStorage`)?

Reply:
Not worth arguing over.  Addressed in 6a95eca9d7dc4f6fe2a6c186ec8d0f85d9fb1f01